File: src/Net/Store/Before/ThirdParty.cs
namespace Before
{
    public class ThirdParty
    {
        public string PhoneNumber { get; private set; }

        public string ContactName { get; set; }

        public ThirdParty(string phoneNumber)
        {
            this.PhoneNumber = phoneNumber;
        }

        public string FormatPhone()
        {
            return string.Format("CountryCode:{0} - Citycode:{1} - LocalNumber:{2}",
                int.Parse(this.PhoneNumber.Substring(0, 2)),
                int.Parse(this.PhoneNumber.Substring(2, 2)),
                int.Parse(this.PhoneNumber.Substring(4)));
        }
    }
}

File: src/Java/2. Second Course/5. Final Hole/src/main/refactoringgolf/store/ThirdParty.java
package refactoringgolf.store;

public class ThirdParty {

	private String phoneNumber;
	private String contactName;

	public ThirdParty(String phoneNumber) {
		this.phoneNumber = phoneNumber;
	}

	public String getPhoneNumber() {
		return phoneNumber;
	}

	public String getContactName() {
		return contactName;
	}

	public String formatPhone() {
		return String.format("CountryCode:%s - Citycode:%s - LocalNumber:%s", phoneNumber.substring(0, 2), phoneNumber.substring(2, 4), phoneNumber.substring(4));
	}
}
